modules/apps/commerce/commerce-inventory-service/src/main/java/com/liferay/commerce/inventory/service/impl/CommerceInventoryWarehouseLocalServiceImpl.java

```diff
@@ -497,14 +507,22 @@ protected void validate(
 		}
 	}
 
+	@Reference
+	protected CommerceInventoryReplenishmentItemLocalService
```

Comment:
@slnn When you add new references, make them private.

Reply:
Update and forced push